12-kubernetes-cluster.tf

```diff
@@ -58,10 +65,15 @@ resource "azurerm_kubernetes_cluster" "kubernetes_cluster" {
   )
 
   identity {
-    type        = "UserAssigned"
+    type         = "UserAssigned"
     identity_ids = [data.azurerm_user_assigned_identity.aks.id]
   }
 
+  kubelet_identity {
+    client_id                 = data.azurerm_user_assigned_identity.kubelet_uami[*].client_id
+    user_assigned_identity_id = data.azurerm_user_assigned_identity.kubelet_uami[*].object_id
```

Comment:
what happens here if you run it with `kubelet_uami_enabled=false`?

Reply:
`data.azurerm_user_assigned_identity.kubelet_uami is empty tuple`